c++/src/kj/compat/tls.c++

```diff
@@ -443,8 +531,31 @@ public:
   }
 
 private:
+  Promise<void> doOneAccept() {
+    // Accept one connection and queue up the next accept on our TaskSet
+
+    return inner->acceptAuthenticated().then(
+        [this](AuthenticatedStream&& stream) {
+          auto promise = kj::evalNow([&] {
```

Comment:
@a-robinson already mentioned the indentation style, but just to add one more that can be odd to newcomers for a while (or was to me): we de-dent the body of the continuation so that it's effectively 2 spaces in from main function body where it's being defined, so this `auto promise` would be de-dented 4 spaces, at which point you may decide to put the lambda capture and parameter list up right next to the `then` -- up to you.

```
return asyncFn().then(
    [cap1,cap2](auto p1, auto p2) {
  auto actualFnBodyHere = foo();
});
```

Or:
```
return asyncFn().then([cap1,cap2](auto p1, auto p2) {
  auto actualFnBodyHere = foo();
});
```

Reply:
Oh man, that has a really negative on my ability to parse the async flow. I'll be doing a lot of trivial movement of lvalue lambdas.

Reply:
I've worked in a codebase with lots of nested async lambdas before. The worst we ever had to do was separate our lambdas into semantic functions if we had too little horizontal space to work with.

For your given example, I'd write this:
```
if (
  really.long.expression.that(
    requires.wrapping.because.its.so.long()
  )
) {
  blockBody();
}
```

I think it boils down to preferring more clearly delineated blocks for me. The sudden left shift indicates a less nested block, which it is not.